app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/view/learning/LearningTest.kt

```diff
@@ -29,10 +28,10 @@ import org.junit.runner.RunWith
 @HiltAndroidTest
```

Comment:
```suggestion
@HiltAndroidTest
@UninstallModules(DatabaseManagementModule::class)
```
(1/2)

Reply:
Thanks a lot!!! It's precisely what I was looking for! And I just tested, it works :D